sample/NormaSample.Web/NormaSample.Db/Scripts/Applications.sql

```diff
@@ -2,8 +2,8 @@
 USING ( VALUES 
 	('DefaultApplication', '1234567890')
 )
-AS Source ([Name], [ApplicationId])
+AS Source ([Name], [ApplicationKey])
 ON Target.[Name] = Source.[Name]
 WHEN NOT MATCHED BY TARGET THEN
-	INSERT ([Name],[ApplicationId])
-	VALUES ([Name],[ApplicationId]);
\ No newline at end of file
+	INSERT ([Name],[ApplicationKey])
```

Comment:
The field name in table Applications is [Key], this file may fail during deployment